config/crd/kustomization.yaml

```diff
@@ -16,7 +16,6 @@ patchesStrategicMerge:
 - patches/webhook_in_machines.yaml
 - patches/webhook_in_machinesets.yaml
 - patches/webhook_in_machinedeployments.yaml
-- patches/webhook_in_machinepools.yaml
```

Comment:
This looks like it's no longer setting up webhooks for the experimental types, is that being handled in a different way?